python/django/containerWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -0,0 +1,42 @@
+"""
+This file demonstrates writing functional tests.
+"""
+import django
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from django.test import TestCase
+import os
+import sys
+import pytest
+
+sys.path.append(os.path.join(os.getcwd(), 'Application'))
+os.environ.setdefault(
+    "DJANGO_SETTINGS_MODULE",
+    "python_webapp_django.settings"
+)
+django.setup()
+
+class FunctionalTests(TestCase):
+
+    def setUp(self):
+        options = webdriver.ChromeOptions()
+        options.add_argument('--no-sandbox')
+        self.driver = webdriver.Chrome(os.environ["ChromeWebDriver"], chrome_options=options)
+
+    def test_selenium(self):
+        try:
+            webAppUrl = pytest.config.getoption('webAppUrl')
+            response = self.driver.get(webAppUrl)
+            html_source = self.driver.page_source
+            self.assertIn("Home Page - Python Django Application", html_source)
```

Comment:
add <title>

Reply:
Why not use self.driver.title ?

---
In reply to: [216280161](https://github.com/Microsoft/devops-project-samples/pull/65#discussion_r216280161) [](ancestors = 216280161)